nautobot/extras/filters.py

```diff
@@ -151,7 +151,7 @@ class CustomFieldChoiceFilterSet(BaseFilterSet):
 
     class Meta:
         model = CustomFieldChoice
-        fields = ["id", "value", "weight"]
+        fields = ["id", "value", "weight", "field"]
```

Comment:
Hmmm. Usually with django_filters it's not necessary or appropriate to include explicitly declared Filters (`field = django_filters.ModelMultipleChoiceFilter`, above) in `fields`:

https://django-filter.readthedocs.io/en/stable/ref/filterset.html#declaring-filterable-fields

> Note that it is *not* necessary to included declared filters in a `fields` list - doing so will have no effect

I'll try and find some time today to play around with this and see if I can figure out why `field` isn't automatically being made available as a filter.

Reply:
Works for me without this change:

![image](https://user-images.githubusercontent.com/5603551/132353966-9fd61f44-cf31-422a-a82a-5dceefbeabfb.png)

![image](https://user-images.githubusercontent.com/5603551/132354090-ae3a52d3-9552-45fd-992a-92e7ec3c3634.png)

Reply:
your are right 😓 , don't why I got the other conclusion